### ISSUE TITLE
No Alt Tag on Alternate product images

### ISSUE THREAD

**Issue by Naraceaus** [NetoECommerce/Skeletal#413]
On the product page there are no alt attributes on the extra product images. This means screen readers won't be able to find them. Even alt="" would be enough for the screen reader to at least know they are there.

**Comment by philcon93** [comment]
This issue is being investigated outside of Github.
